### DISCUSSION TITLE
Telescope Control Issue

### DISCUSSION THREAD

**Discussion by TonyMan13** [Stellarium/stellarium#3264] [Q&A]
My club recently retrofitted one of our observatory scopes with new cameras and wanted to use planetarium software to help operators use it. The scope is on a clock driven equatorial fork mount. We’ve installed one of Dave Ek’s setting circles boards and we seem to be on the right path, but are kinda stumped. After alignment we got to another object. As time goes by, the image stays centered in the camera software, but the object drifts from the crosshairs representing the scope in Stellarium due to earths rotation. One member feels there should be a setting in Stellarium or the ASCOM that tells the software the scope is clock driven to resolve this. We’ve looked around , but don’t see anything that looks helpful. Are we missing something, or are we expecting too much?
 
Any help is appreciated.

**Comment by gzotti** [comment]
I am afraid none of the current Stellarium developers know what a "Dave Ek's setting circles board" is. If this is supposed to make an LX200 or ASCOM-compatible scope out of your existing vintage telescope, maybe it is up to their support to answer this question. 

Given that we had reports (see FAQ) about issues with the TelescopeControl plugin of the Qt6-based builds with some telescopes, be sure to try with a Qt5-based build ("qt5" or "qt6" are part of the installer filename). We need a developer-observer with such a telescope to finally identify and fix those issues. Reward possible!

**Comment by TonyMan13** [comment]
I only included the info on the Dave Ek board to see if it was helpful. Basically, it can be replaced by saying we've got the setting circles connected via ASCOM. The basic issue is, after alignment, we slowly push the scope to a new object. On Stellarium we see the reticle move to the object, which leads us to believe things are pretty much working. However, as the clock drive (motor) keeps the object centered in our camera, Stellarium shows the object's motion, but the reticle stays where we moved it to. Since it followed the original scope movement we thought the reticle would stay on the object. Just wondering if there is something we missed. 

In the meantime , I'll check out the faqs you mentioned.

**Reply by gzotti** [reply]
Not directly, although with a product name is better than without. The discussion on CloudyNight revealed that it might be a push-to telescope. Indeed, this would mean the position feedback is missing.
